.github/drafts/16506/16506_range_slider_outline.md

```diff
@@ -0,0 +1,76 @@
+*** Range Slider ***
+
+The Range Slider widget allows users to choose a range of values from numerical data.
+
+< TODO: NEED VIDEO DEMO >
+
+---
+# Usage
+- Example use case: Search filtering (i.e. Min/ Max price)
+
+---
+
+# Properties
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+## Widget Properties
+
+These properties allow you to edit the Range Slider widget. The following table lists all the widget's unique properties; if you do not see a certain widget property here, it may be listed under [Common Widget Properties]().
+
+| **Property** | **Description** |
+|--------------|-----------------|
+| **Min Value** | The starting/smallest possible value of the slider. The value may be negative and/or include decimals, however it must always be less than the **Max Value**. _(number)_ |
+| **Max Value** | The ending/largest possible value of the slider. The value may be negative and/or include decimals, however it must always be greater than the **Min Value**. _(number)_ |
+| **Step Size** | The increment by which the user can adjust the slider's value. This increment must be at least 0.1, and cannot be less than the **Min Value** or greater than the **Max Value**. _(number)_ |
+| **Min Range** |  |
+| **Default Start Value** |  |
+| **Marks** | An _array of objects_ with keys `value` and `label` which define where reference labels should appear below the widget's slider. These labels can be shown or hidden with the **Show Marks** property toggle. |
+| **Tooltip Always On** | Hovering over the slider with the mouse cursor shows the slider's selected value in a tooltip; enabling this setting will force the tooltip to always be visible, regardless of the cursor's location. _(bool)_ |
+
+### Min Value / Max Value
+
+### Step Size
+
+### Min Range
+
+### Default Start Value
+
+### Marks
+
+### Tooltip Always On
+
+## Binding Properties
```

Comment:
Please update based on the [widget template](https://www.notion.so/appsmith/Widget-Doc-template-5d7fa61a2e2a4ee888487ba9b43bf487#6df273cbb4e84edb94931b6e0bc59de1).